### PR TITLE
Fix YARD documentation style

### PR DESCRIPTION
**Pull request by mbj**

### CODE REVIEW

**Comment by mbj** [comment]
@dkubb Assuming all green this is ready for review. Please review / merge #359 before this one.

**Comment by dkubb** [comment]
So far the YARD changes look fine, but it's hard to see all of them until #359 is merged in.

**Comment by mbj** [comment]
@dkubb thx for rebasing.

File: lib/mutant/ast/meta.rb [hunk 1]
```diff
@@ -19,12 +19,11 @@ class Send
         INDEX_ASSIGNMENT_SELECTOR            = :[]=
         ATTRIBUTE_ASSIGNMENT_SELECTOR_SUFFIX = '='.freeze
 
-        # Return arguments
+        # The arguments of node
```
Comment (dkubb) [inline lib/mutant/ast/meta.rb]: What about `The arguments for node` ?

Comment (mbj) [inline lib/mutant/ast/meta.rb]: For my limited English this carries the same information. But I'll simply use your suggestion as its likely your suggestion is what I actually meant.

File: lib/mutant/env/bootstrap.rb [hunk 2]
```diff
@@ -75,7 +70,7 @@ def env
 
     private
 
-      # Return scope name
+      # The scope name from scopeing object
```
Comment (dkubb) [inline lib/mutant/env/bootstrap.rb]: There is a spelling mistake here.

File: lib/mutant/mutator/node/arguments.rb [hunk 3]
```diff
@@ -79,12 +74,11 @@ def emit_mlhs_expansion
           end
         end
 
-        # Return mlhs childs
+        # The mlhs childs
```
Comment (dkubb) [inline lib/mutant/mutator/node/arguments.rb]: What does mlhs stand for? If it's an uncommon abbreviation I would generally expand it in the documentation.

File: lib/mutant/mutator/node/send.rb [hunk 4]
```diff
@@ -66,20 +64,21 @@ def non_index_dispatch
...
         # @return [AST::Meta::Send]
+        #
+        # @api private
         def meta
           AST::Meta::Send.new(node)
         end
         memoize :meta
 
-        # Return arguments
+        # The arguments being send
```
Comment (dkubb) [inline lib/mutant/mutator/node/send.rb]: Do you mean `sent` ?

File: lib/mutant/mutator/node/when.rb [hunk 5]
```diff
@@ -57,17 +54,15 @@ def mutate_body
         #   otherwise
         #
         # @api private
-        #
         def body
           children[body_index]
         end
 
-        # Return body index
+        # The index of body node
```
Comment (dkubb) [inline lib/mutant/mutator/node/when.rb]: what about `... the body node` ?

File: lib/mutant/reporter/cli/printer/env_progress.rb [hunk 6]
```diff
@@ -39,22 +38,20 @@ def run
 
         private
 
-          # Return coverage percent
+          # The coverage in percent
```
Comment (dkubb) [inline lib/mutant/reporter/cli/printer/env_progress.rb]: What about `The coverage percentage` ?

File: lib/mutant/reporter/cli/printer/env_progress.rb [hunk 7]
```diff
@@ -39,22 +38,20 @@ def run
...
           # @return [Float]
           #
           # @api private
-          #
           def coverage_percent
             coverage * 100
           end
 
-          # Return overhead percent
+          # The overhead in percent
```
Comment (dkubb) [inline lib/mutant/reporter/cli/printer/env_progress.rb]: What about `The overhead percentage` ?

File: lib/mutant/reporter/cli/tput.rb [hunk 8]
```diff
@@ -7,13 +7,15 @@ class Tput
 
         private_class_method :new
 
-        # Return detected tput support
+        # The detected tput support
```
Comment (dkubb) [inline lib/mutant/reporter/cli/tput.rb]: I wonder if many cases where you used `The` as a prefix on the method description could simply be removed without affecting comprehensibility?

A method named `self.detect` with a description `Detected tput support` seems just as reasonable as this.

File: lib/mutant/mutator/node/arguments.rb (hunk 3 above)
Comment (mbj) [inline lib/mutant/mutator/node/arguments.rb]: Multiple  left hand side. For people dealing with the ruby AST its more common https://github.com/whitequark/parser/blob/master/doc/AST_FORMAT.md#multiple-left-hand-side. I think expanding it makes sense.

**Comment by dkubb** [comment]
@mbj I left a few notes.

My largest observation is that maybe the `The` prefix on the description is unnecessary in most cases. It's just extra boilerplate and could probably be removed without making things harder to understand.

**Comment by mbj** [comment]
> My largest observation is that maybe the The prefix on the description is unnecessary in most cases. It's just extra boilerplate and could probably be removed without making things harder to understand.

I was trying to give all getters the same prefix. I think that failed. I'll kill them.

**Comment by mbj** [comment]
@dkubb Ready for your next pass.

**Comment by tjchambers** [comment]
@dkubb It looks as though in this update you were striving to remove the word "Return" prefixing the descriptions in the comments. You did not get all of them. Would you like me to make a pass?

**Comment by mbj** [comment]
> @dkubb It looks as though in this update you were striving to remove the word "Return" prefixing the descriptions in the comments. You did not get all of them. Would you like me to make a pass?

Ideally do a pull request fixing them. This PR is merged a while now and adding flags here is not perfect. BTW in some situartions "return" might make sense. But could be also an artifact from my limited English.

**Comment by tjchambers** [comment]
I will take that as a task over the weekend as a PR and them you can comment for my adjustment.

**Comment by mbj** [comment]
@tjchambers cool, thx.

**Comment by tjchambers** [comment]
I went through the whole repo as it is currently and only found one of these, and I would leave it as is, so unfortunately I have nothing to contribute.

**Pull request merged**